packages/react/src/components/Forms/Select/mono-common.tsx

```diff
@@ -174,27 +174,35 @@ export const SelectMonoAux = ({
           {...downshiftReturn.getMenuProps()}
         >
           <ul>
-            {downshiftReturn.isOpen &&
-              options.map((item, index) => {
-                const isActive = index === downshiftReturn.highlightedIndex;
-                return (
-                  <li
-                    className={classNames("c__select__menu__item", {
-                      "c__select__menu__item--highlight": isActive,
-                      "c__select__menu__item--selected":
-                        downshiftReturn.selectedItem === item,
-                      "c__select__menu__item--disabled": item.disabled,
-                    })}
-                    key={`${item.value}${index}`}
-                    {...downshiftReturn.getItemProps({
-                      item,
-                      index,
-                    })}
-                  >
-                    <span>{item.label}</span>
+            {downshiftReturn.isOpen && (
+              <>
+                {options.map((item, index) => {
+                  const isActive = index === downshiftReturn.highlightedIndex;
+                  return (
+                    <li
+                      className={classNames("c__select__menu__item", {
+                        "c__select__menu__item--highlight": isActive,
+                        "c__select__menu__item--selected":
+                          downshiftReturn.selectedItem === item,
+                        "c__select__menu__item--disabled": item.disabled,
+                      })}
+                      key={`${item.value}${index}`}
```

Comment:
Eslint is not happy: _Do not use Array index in keyseslint[react/no-array-index-key](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-array-index-key.md)_

`item.value` can be undefined, what about that (toString is to not get the eslint warning)? 
```suggestion
                      key={`${item.label}${index..toString()}`}
```

Reply:
I agree but this is the way it is demonstrated in the example on the doc: https://www.downshift-js.com/use-select

I guess that what they want is to clear the component if the index changes too. But I understand that it seems odd ^^' 

I update to use `toString` but I keep in mind that there are other warnings that need to be fixed in the near future in another PR.

Reply:
Yes but a doc can be wrong or has to be adapted for your own case. In this case, if value is `undefined`, the keys will be `["undefined1", "undefined2", ...]`.

Reply:
Sure but here appart the value that could be undefined ( I will update this ) we are in the exact same use case as the documentation which is weird regarding including `index` .. 

Anyway I will deal with it in a separate PR.